breathecode/certificate/models.py

```diff
@@ -25,6 +25,7 @@ class Meta:
 
 # For example: Full-Stack Web Development
 class Specialty(models.Model):
+    academy = models.ForeignKey(Academy, on_delete=models.CASCADE, related_name="specialties", null=True, blank=True)
```

Comment:
@tommygonzaleza do you want to have specialties linked to an academy?

Reply:
https://github.com/orgs/breatheco-de/projects/7/views/1?filterQuery=avanegas&pane=issue&itemId=96008958&issue=breatheco-de%7Cbreatheco-de%7C8368